bot.js

```diff
@@ -30,6 +32,19 @@ rtm.on(RTM_EVENTS.MESSAGE, function handleRtmMessage(message){
     var channelId = message.channel;
 
     if(message.text && message.text.indexOf(botId) > - 1){
+        var user = message.text.replace(/ /g, '').replace(botId, '').replace(':', '');
+            var regex = '^<@\..*>$';
+
+        if(user.match(query)){
```

Comment:
query is not defined here?  Did you mean to use Regex?  If I use regex, the if statement never seems to be triggered.